PhysicsTools/PatAlgos/python/producersLayer1/tauProducer_cff.py

```diff
@@ -20,7 +20,7 @@
 
 makePatTausTask = cms.Task(
     # reco pre-production
-    patHPSPFTauDiscriminationTask,
+    #patHPSPFTauDiscriminationTask,
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Same for the older patTauJetCorrections below